krave/output/data_writer.py

```diff
@@ -62,7 +62,7 @@ def update_meta(self, session_data):
     def make_events(self):
         events_path = os.path.join(self.data_write_path, self.events_name)
 
-        with open(PATHS.INITIAL_COMMUNICATION, 'a') as file:
+        with open(PATHS.COMMUNICATION, 'a') as file:
```

Comment:
For future improvements, maybe we could consider making all cross communication functions behind a class. It can be a bit confusing sometimes to write directly to files everywhere in the code. Don't fix it now though, it's fine to merge as is